newrelic.yml

```diff
@@ -175,6 +175,9 @@ common: &default_settings
   # If true, disables Active Storage instrumentation.
   # disable_active_storage: false
 
+  # If true, disables Active Support instrumentation.
```

Comment:
`default_source.rb` uses `ActiveSupport` for the description but `newrelic.yml` here is using `Active Support`.

Reply:
Made consistent in: 278635542852a978fcb57364752500c445f237c6